src/vorta/assets/exclusion_presets/browsers.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "name": "Chromium cache and config files",
+        "slug": "chromium-cache",
+        "patterns":
+        [
+            "*.config/chromium/*/Local Storage",
+            "*.config/chromium/*/Session Storage",
+            "*.config/chromium/*/Service Worker/CacheStorage",
+            "*.config/chromium/*/Application Cache",
+            "*.config/chromium/*/History Index *",
+            "snap/chromium/common/.cache",
+            "snap/chromium/*/.config/chromium/*/Service Worker/CacheStorage",
+            "snap/chromium/*/.local/share/"
+        ],
+        "tags":["application:chromium", "type:browser", "os:linux"],
+        "author": "Divi"
```

Comment:
Why did we keep the author field? Authorship is already tracked in git, we don't need to add an unused field that has to be updated when changing a preset.

Reply:
I don't mind having the ownership outside Github/git. My hope is that people will add their presets and maybe maintain them.